### DISCUSSION TITLE
does Webmin work on Windows and so yes how to install ?

### DISCUSSION THREAD

**Discussion by ned-qua** [webmin/webmin#1878] [Q&A]
I am looking for a hosting panel that would work on Linux and Windows other than cPanel or Plesk… I found webmin, and it looks like a really great package, but I would like to install and configure it on Windows servers as well. Is that possible and so yes, how to install and to configure?

**Comment by chris001** [comment]
Yes, there's a step by step to install Webmin on Windows:
https://doxfer.webmin.com/Webmin/Installing_Webmin_on_Windows

**Reply by jcameron** [reply]
Webmin is really not well supported on Windows though ... it can be made to work, but I wouldn't recommend it!

**Comment by swelljoe** [comment]
The only people who were maintaining docs and testing Webmin on Windows no longer do so, as far as I know, and haven't in years. So, I would guess it doesn't work very well on Windows at this point.

Also, Webmin by itself isn't a web hosting control panel. You'd also need Virtualmin, which definitely doesn't work on Windows (though it does reportedly work in WSL, the "Linux in Windows" implementation of Ubuntu). I haven't tested that, personally, but people have posted about using it without much difficult. But, if you're running on Ubuntu in Windows, you might as well just run Ubuntu natively and save a license fee.
